data/transactions/actions.ts

```diff
@@ -19,6 +19,8 @@ import { Transaction } from "./reducer";
 import { transactionsLatestBlockSelector } from "../selectors";
 import { transactionsSelector } from "./selectors";
 
+const appendBCHPrefix = (target: string) => `bitcoincash:${target}`;
```

Comment:
suggestion: make a function that you can apply on all address strings and will add the prefix if its missing.

some pseudo logic

  let split = string.split(':')[1];

    if (split === undefined) {
   //    
  }

Reply:
Noted, good suggestion.
Let's improve this in the next PR.